NEWS.md

```diff
@@ -2,8 +2,8 @@
 
 NEW FUNCTIONS
 
-* `rowmean_n()`, to compute row means if row contains at least `n` non-missing
-  values.
+* `row_means()`, to compute row means, optionally only for those row with at
```

Comment:
why is it plural mean*s* ? doesn't it return just *the* mean?

Reply:
Like `rowMeans()` from base R, it computes mean_s_ of several rows.